data-8/jupyterhub-k8s
# Allow kubernetes secrets to be used

Closes #163

Summary
-------
Updated the hub.yml, proxy.yml and cull.yml chart to allow for the deployment to use kubernetes secrets when secrets and token are not included in the config.yml file.
Updated the values.yml to have null values for lookup purposes.

How to test
-------
1: Remove token/cookie secret from config file
2: Create a kubernetes secret name 'hub-secret' with the same keys as would be in the ConfigMap
3: Upload secret file
4: Run `helm upgrade` to get new config
5: Restart Pods if necessary
6: Check environment variable on running pods to ensure they are running correctly
